app/helpers/check_answers_helper.rb

```diff
@@ -38,6 +39,25 @@ def question_items(question)
     end
   end
 
+  def counts_key(name)
+    "#{name.downcase.gsub(' ', '_')}_number".to_sym
+  end
+
+  def humanize_count(count, postfix)
+    number = number_with_delimiter(count, delimiter: ",")
+    pluralize(number, postfix)
+  end
+
+  def create_list_with_named_values(names, counts, postfix)
```

Comment:
```suggestion
  def create_list_with_named_values(items, counts, postfix)
```
This is not a blocker, but `names` brings to mind "first name", "last name" etc, rather than being generic as I think this method is trying to be.

Reply:
Thanks @leenagupte 👍  Outdated - resolved.